kth-largest-element-in-an-array.py

```diff
@@ -0,0 +1,18 @@
+'''
+- Time Complexity: O(N + klogN)
+- Space Complexity: O(1) - takes k spaces
```

Comment:
Please specify what `N` and `K` indicates.

Reply:
Thanks for verifying updated N and k